src/lib.rs

```diff
@@ -167,7 +167,9 @@ mod builder;
 pub use builder::{KnownHosts, SessionBuilder};
 
 mod command;
-pub use command::Command;
+pub use command::{Command, OverSsh};
+
+pub(crate) mod escape;
```

Comment:
```suggestion
mod escape;
```

`lib.rs` is the top-level of a crate, any definition here we be available to all submodules.